app/assets/javascripts/datarepo/funder.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function(){
+
+  $('.collection_funder').find('.collection_funder').each( function(index, value) {
```

Comment:
If you add a class to the edit_field partial, we can reuse this code, if we need to.